config/crd/bases/activemonitor.keikoproj.io_healthchecks.yaml

```diff
@@ -203,6 +203,9 @@ spec:
               type: integer
             totalHealthCheckRuns:
               type: integer
+          required:
```

Comment:
Please check this.. they needed not be in required. Update latest crd and try.

Reply:
updated CRD and verified. Looking good.